inc/zoo/swar/SWAR.h

```diff
@@ -77,6 +77,20 @@ struct SWAR {
 
     constexpr T value() const noexcept { return m_v; }
 
+    template<std::size_t N, typename = std::enable_if_t<N == Lanes, T>>
+    constexpr static T baseFromLaneLiterals(const T (&args)[N]) {
```

Comment:
Raises the follow-up: Do you think `BooleanSWAR(4,u16)::fromLaneLiterals(true, false, true, fasel)` would be useful?  I acknowledge the point you made the other day that most engineers aren't and will not be fluent in hexadecimal.

Reply:
Yes, let's do it, I'm "convinced" now:
```
SWAR my4x16{Literals<16, uint64_t>, {0xFF00, 0xF0F0, 0xFFFF, 0xF000}};
BooleanSWAR{Literals<16, uint64_t>, {true, true, false, true}};
```

Reply:
ok i follow.